dual_net.py

```diff
@@ -206,8 +206,8 @@ def my_res_layer(inputs):
     # train ops
     global_step = tf.train.get_or_create_global_step()
     policy_cost = tf.reduce_mean(
-        tf.nn.softmax_cross_entropy_with_logits(
-            logits=logits, labels=labels['pi_tensor']))
+        tf.nn.softmax_cross_entropy_with_logits_v2(
+            logits=logits, labels=tf.stop_gradient(labels['pi_tensor'])))
```

Comment:
This `tf.stop_gradient` may not be necessary, but I added it just to be safe.

See https://www.tensorflow.org/api_docs/python/tf/nn/softmax_cross_entropy_with_logits and https://www.tensorflow.org/api_docs/python/tf/nn/softmax_cross_entropy_with_logits_v2 for context.

Reply:
Can you back this out and put it in another PR?  I'm happy to have it would just like to avoid changing the core net code in a PR for an analysis script :)

Reply:
That's fair--done.